block_ctx

```diff
@@ -0,0 +1,883 @@
+BlockContext {
```

Comment:
is this file included accidentally?

Reply:
yeah, sorry accidentally added some test files from the spirv parser PR